src/main/webapp/app/tasks/local-task.service.ts

```diff
@@ -34,7 +34,20 @@ export class LocalTaskService implements TaskService {
 
   private readTasks(): Task[] {
     const tasks = localStorage.getItem(LocalTaskService.STORAGE_KEY);
-    return tasks ? JSON.parse(tasks) : [];
+    const taskJson = JSON.parse(tasks); 
+    taskJson.sort(comp);
```

Comment:
I'd prefer an inline function for this. But more important: This is some newly added functionality which should be tested.